From 329e71443e65a287996bf161879d4ffd224de09f Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas@nowhere.net>
Date: Wed, 7 Oct 2015 19:45:36 +0200
Subject: [PATCH] tools: remove configs symlink.

dh_python3 crashes if there is a symlink in the site-dist/lirc
directory, see Debian BTS #801719. Link restored in rules.
---
 tools/Makefile.am | 6 ------
 1 file changed, 6 deletions(-)

--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -121,12 +121,6 @@
 	    $(DESTDIR)/$(pkgpythondir)/lirc-setup \
 	    $(DESTDIR)/$(bindir)/lirc-setup
 
-install-data-hook:
-	rm -f $(DESTDIR)/$(pkgpythondir)/configs
-	$(srcdir)/make_rel_symlink.py \
-	    $(DESTDIR)$(pkgdatadir)/configs \
-	    $(DESTDIR)$(pkgpythondir)/configs
-
 uninstall-hook:
 	rm -f $(DESTDIR)/$(bindir)/lirc-setup \
 	      $(DESTDIR)/$(pkgpythondir)/configs
